[master] Fails to build with error: No rule to make target 'sm.png', needed by 'all-am'.  Stop.

Apparantly caused by commit 7ac5c5cd5304dcf462c0143d6273d4bd4426bbc1:

```console
# autoreconf -i -f
# mkdir build
# cd build
# ../configure && make
[..]
make  all-am
make[1]: Entering directory '[..]/screen-message-fork/build'
gcc -DHAVE_CONFIG_H -I. -I..    -I/usr/include/gtk-3.0 -I/usr/include/atk-1.0 -I/usr/include/pango-1.0 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/cairo -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/elogind -I/usr/include/webp -I/usr/include/gio-unix-2.0 -I/usr/lib64/libffi/include -pthread -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/libpng16 -I/usr/include/pixman-1 -g -O2 -MT sm-sm.o -MD -MP -MF .deps/sm-sm.Tpo -c -o sm-sm.o `test -f 'sm.c' || echo '../'`sm.c
mv -f .deps/sm-sm.Tpo .deps/sm-sm.Po
gcc -I/usr/include/gtk-3.0 -I/usr/include/atk-1.0 -I/usr/include/pango-1.0 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/cairo -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/elogind -I/usr/include/webp -I/usr/include/gio-unix-2.0 -I/usr/lib64/libffi/include -pthread -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/libpng16 -I/usr/include/pixman-1 -g -O2   -o sm sm-sm.o -lgtk-3 -latk-1.0 -lgio-2.0 -lgdk-3 -lpangocairo-1.0 -lgdk_pixbuf-2.0 -lcairo-gobject -lcairo -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lharfbuzz  
sed -e s%@BINPATH@%/usr/local/games% <../sm.desktop.in >sm.desktop
make[1]: *** No rule to make target 'sm.png', needed by 'all-am'.  Stop.
make[1]: Leaving directory '[..]/screen-message-fork/build'
make: *** [Makefile:346: all] Error 2
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -13,7 +13,7 @@ desktopdir = $(datadir)/applications
 desktop_DATA = sm.desktop
 
 icondir = $(datadir)/icons/hicolor/48x48/apps
-dist_icon_DATA = sm.png
+dist_icon_DATA = docs/sm.png
 
 gtk_update_icon_cache = gtk-update-icon-cache -f -t $(datadir)/icons/hicolor
 
@@ -31,7 +31,7 @@ update-icon-cache:
 dist_man_MANS = sm.6
 # some of these files are just added to make the Debian packaging based on the
 # same VCS painless... (otherwise dpkg would try to add these files as patches)
-EXTRA_DIST = sm.py sm.desktop.in sm.html sm.webapp webapp.html README.Win32 sm.ico sm.rc sm-128.png sm.svg gtkzoom.h gtkzoom.c sm-fsmi-ka.jpg
+EXTRA_DIST = sm.py sm.desktop.in docs/index.html docs/sm.webapp docs/webapp.html README.Win32 sm.ico sm.rc docs/sm-128.png docs/sm.svg gtkzoom.h gtkzoom.c docs/sm-fsmi-ka.jpg
 CLEANFILES = sm.desktop 
 
 if WIN32

